contracts/weapons.sol

```diff
@@ -6,17 +6,13 @@ import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol"
 import "../node_modules/@openzeppelin/contracts/math/SafeMath.sol";
 import "../node_modules/abdk-libraries-solidity/ABDKMath64x64.sol";
 import "./util.sol";
-import "./interfaces/ITransferCooldownable.sol";
 
-contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable, ITransferCooldownable {
+contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable {
 
     using ABDKMath64x64 for int128;
     using ABDKMath64x64 for uint16;
 
     bytes32 public constant GAME_ADMIN = keccak256("GAME_ADMIN");
-    bytes32 public constant RECEIVE_DOES_NOT_SET_TRANSFER_TIMESTAMP = keccak256("RECEIVE_DOES_NOT_SET_TRANSFER_TIMESTAMP");
```

Comment:
I would like to keep this constant for now, as it is used in `migrations/29_transfer_cooldown.js`. While that is an old migration, and this constant won't actually be applicable in any way after this PR is merged, I would like to leave all applied migration files untouched for now.

Eventually I would like to merge all the old applied migrations we have right now into a single file - I think that would be a good time to finally remove this constant.